index.html

```diff
@@ -1676,15 +1697,19 @@ <h2 id="AudioContext">
               <li>Set a <code>control thread state</code> to
               <code>suspended</code> on the <a>AudioContext</a>.
               </li>
-              <li>Send a <a>control message</a> to start processing.
+              <li>Let <dfn>pendingResumePromises</dfn> be an empty ordered list
```

Comment:
Yes, because we don't store them for later use, we immediately return them to content.

Here, we need to keep them around to resolve or reject them at a later date. It's necessary to store them somewhere to do that.